### PR TITLE
New package: caps-0.9.26

### PR DESCRIPTION
**Pull request by laserbat**

### CODE REVIEW

File: srcpkgs/caps/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+makedepends="ladspa-sdk"
+short_desc="The LADSPA C* Audio Plugin Suite"
+maintainer="Olga Ustuzhanina <me@laserbat.pw>"
+license="GPL-3.0-or-later"
+homepage="http://quitte.de/dsp/caps.html"
+distfiles="http://quitte.de/dsp/${pkgname}_${version}.tar.bz2"
+checksum=e7496c5bce05abebe3dcb635926153bbb58a9337a6e423f048d3b61d8a4f98c9
+
+do_build() {
+	make ${makejobs}
```
Comment (ndowens) [inline srcpkgs/caps/template]: Might could put ${makejobs} in make_build_args

File: srcpkgs/caps/template [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+revision=1
+build_style=gnu-makefile
+makedepends="ladspa-sdk"
+short_desc="The LADSPA C* Audio Plugin Suite"
+maintainer="Olga Ustuzhanina <me@laserbat.pw>"
+license="GPL-3.0-or-later"
+homepage="http://quitte.de/dsp/caps.html"
+distfiles="http://quitte.de/dsp/${pkgname}_${version}.tar.bz2"
+checksum=e7496c5bce05abebe3dcb635926153bbb58a9337a6e423f048d3b61d8a4f98c9
+make_use_env=yes
```
Comment (Johnnynator) [inline srcpkgs/caps/template]: move `make_use_env=yes` after build style and add `make_build_args='CC=$(CXX)'` in the line after it.

**Comment by Johnnynator** [comment]
```
--- basics.h	2019-01-07 16:13:33.353999394 +0100
+++ basics.h	2019-01-07 16:13:44.915942123 +0100
@@ -46,17 +46,18 @@
 
 #include <assert.h>
 #include <stdio.h>
+#include <stdint.h>
 
 #include "ladspa.h"
 
-typedef __int8_t			int8;
-typedef __uint8_t			uint8;
-typedef __int16_t			int16;
-typedef __uint16_t		uint16;
-typedef __int32_t			int32;
-typedef __uint32_t		uint32;
-typedef __int64_t			int64;
-typedef __uint64_t		uint64;
+typedef int8_t			int8;
+typedef uint8_t			uint8;
+typedef int16_t			int16;
+typedef uint16_t		uint16;
+typedef int32_t			int32;
+typedef uint32_t		uint32;
+typedef int64_t			int64;
+typedef uint64_t		uint64;
 
 #define MIN_GAIN 1e-6 /* -120 dB */
 /* smallest non-denormal 32 bit IEEE float is 1.18e-38 */
```
This patch should get it working on musl.

**Comment by laserbat** [comment]
Done! Thank you so much for your help @Johnnynator, I'm really sorry for taking your time with my PRs

**Pull request merged**